portal/webpack.common.config.js

```diff
@@ -7,21 +9,31 @@ module.exports = {
     mojFrontend: './scripts/moj-frontend.js',
     maskedInputs: './scripts/masked-inputs.js',
     jsEnabled: './scripts/js-enabled.js',
+    fileUpload: './scripts/file-upload.js',
+    correspondenceAddress: './scripts/correspondence-address.js',
   },
   output: {
     path: path.join(__dirname, 'public/js'),
     filename: '[name].js',
-    library: ['DTFS', '[name]'],
+    library: ['[name]'],
```

Comment:
Pretty sure this change will break some BSS things, there's some client side JS that relies on DTFS document object